apiData_arff.js

```diff
@@ -43,10 +43,16 @@ function jsonToArff(json_data, classKey, fileName, destination) {
         var element = json_data[i];
         for (var key in element) {
```

Comment:
we need to sort the key alphabetically, as the the JSON from the API returns Pokemon sights which have a different order. e.g. once pokemonId is the first key and in another sample, it is the third key..
If you check out the .arff file you can see that at some point the data in a column changes, e.g. from source name to pokemonId. As a result, Weka cannot open the .arff file